buildchow/todo.adoc

```diff
@@ -0,0 +1,69 @@
+= Advantages noticed within a week:
+
+* fixing JRE across all jobs
+* relasing target platform without breaking builds
+* using 9.0 instead of 4.3.mars for stream
+
+= Local Jenkins Configuration
+
+Things I needed to do to run/test on local Jenkins (not complete yet!)
+
+. Add label to have it be recognized as a runnable node. (i.e. RHEL7)
```

Comment:
better yet, parameterize that at the server level so that for your Jenkins you can  simply set label = ""; for JBoss Jenkins, you use the full one with RHEL6/RHEL7/etc.